mobile/Ihc/components/MedicationInventory.js

```diff
@@ -16,19 +16,86 @@ export default class MedicationInventory extends Component<{}> {
    *  {
    *    rows: [Medication],
    *    createMedication: function,
-   *    updateMedication: function
+   *    updateMedication: function,
+   *    deleteMedication: function
    *  }
    */
 
   constructor(props) {
     super(props);
-    this.tableHeaders = ['Drug Name', 'Quantity', 'Dosage', 'Units', 'Notes'];
+    this.tableHeaders = ['Drug Name', 'Quantity', 'Dosage', 'Units', 'Notes', ' ']; //blank header for 'x' column
     this.rowNum = 0;
 
     const formValues = {drugName: null, quantity: null, dosage: null, units: null, comments: null};
     this.state = { showModal: false, medicationKey: null, formOptions: this.addModalFormOptions, formValues: formValues};
   }
 
+  getStyle(index) {
+    switch(index) {
+      case 0:
+        return styles.drugNameCol;
+      case 1:
+      case 2:
+      case 3:
+        return styles.otherCol;
+      case 4:
+        return styles.notesCol;
+      case 5:
+        return styles.emptyCol;
+      default:
+        return styles.otherCol;
+    }
+  }
+
+  getSize(index) {
+    switch(index) {
+      case 0: // drug name
+        return 3;
+      case 1: // quantity
+      case 2: // dosage
+      case 3: // units
+        return 1;
+      case 4: // notes
+        return 3;
+      default:
+        return 1;
+    }
+  }
+
+  getText(index) {
```

Comment:
nit: can we rename this as getTextStyle? It's a little confusing that you are retrieving styles for a getText